src/baseTrie.js

```diff
@@ -434,7 +467,8 @@ module.exports = class Trie {
           childKey.push(childIndex)
           const priority = childKey.length
           taskExecutor.execute(priority, taskCallback => {
-            self._lookupNode(childRef, childNode => {
+            self._lookupNode(childRef, (e, childNode) => {
+              if(e){ return cb(e, null)}
```

Comment:
This is the line that makes my tests pass, but I added the line to anywhere `_lookupNode` is being called (except in `checkRoot` which I believe is expected to just return false in that case).

So probably it could use some more tests that make sure it's properly propagating the other errors (but current tests continue to pass).